tensorboard/plugins/projector/projector_plugin_test.py

```diff
@@ -193,13 +227,39 @@ def testPluginIsActive(self):
     self._GenerateProjectorTestData()
     self._SetupWSGIApp()
 
-    # Embedding data is available.
+    # The is_active method makes use of a separate thread, so we mock threading
+    # behavior to make this test deterministic.
+    tf.test.mock.patch('threading.Thread', FakeThread).start()
```

Comment:
Yes: call `thread.run()`, as in the snippet above. This will invoke the thread's handler synchronously.